pkg/build/controller/strategy/util.go

```diff
@@ -32,6 +32,10 @@ const (
 	ConfigMapCertsMountPath              = "/var/run/configs/openshift.io/certs"
 	SecretBuildSourceBaseMountPath       = "/var/run/secrets/openshift.io/build"
 	SourceImagePullSecretMountPath       = "/var/run/secrets/openshift.io/source-image"
+	//TODO Ben and Adam were discussing this in https://github.com/openshift/cluster-image-registry-operator/pull/340
+	//with Adam noting he asked for clarification in the proxy design doc; a little curious if this should be a
+	//directory to hold multiple pem files, or will this contain an aggregated item
+	ConfigMapBuildProxyCAMountPath = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
```

Comment:
Per arch call this is the correct mount path, but we should mount to a directory (not an individual file).

```suggestion
	ConfigMapBuildProxyCAMountPath = "/etc/pki/ca-trust/extracted/pem/"
```

Reply:
part of next push